### ISSUE TITLE
The objective functions in the code

### ISSUE THREAD

**Issue by Hazeliii** [adosum/3D-FlowNet#2]
Hi Adosum. Thanks for you release.

I am confused about the objective functions in your code. For example , the _zhu_timestamp_objective_ in_ _utils/event_utils.py_, it is written as:

```
loss = -(np.sum(posimg*posimg)+np.sum(negimg*negimg))
 return loss
```

In the paper  _Unsupervised Event-based Learning of Optical Flow, Depth, and Egomotion_ by Zhu et al,  it said that we should minimize the loss function when using it as the loss function of a deep learning network.  But here you used the negative.

So I want to know that in your code, will the optimizer maximize or minimize the objective functions?

**Comment by adosum** [comment]
Hi, sorry for the bad arrangement of my code. That function is for testing other's project and I forgot to delete it from my code. Check this link for the project https://github.com/TimoStoff/events_contrast_maximization.

And I realized that this loss funtion is more suitable for the global optical flow(check the mvsec dataset, the ground truth is actually the global flow), and is less stable than the loss function i am using. Correct me if i am wrong about this.
